debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugRunner.kt

```diff
@@ -146,6 +146,8 @@ private fun buildProjectAndGetBinaryArtifactPath(project: Project, command: Carg
                             .mapNotNull { CargoMetadata.Artifact.fromJson(it) }
                             .filter { (target, profile) ->
                                 target.cleanKind == CargoWorkspace.TargetKind.BIN
+                                    // TODO: support cases when `crate_types` contains not only binary
+                                    || target.cleanKind == CargoWorkspace.TargetKind.EXAMPLE && target.crate_types.singleOrNull() == "bin"
```

Comment:
`target.crate_types.singleOrNull() == "bin"` does not look too great :) 

Let's add an enum for this, sort-of like `cleanKind` works, and also add a comment with a link to the relevant comment in Cargo, which explains what's going on. 

Other than this stylistic drawback, r+!